### ISSUE TITLE
big differences in versions?

### ISSUE THREAD

**Issue by yeahman45** [Diaoul/subliminal#1030]
I don't understand but almost all the options for CLI that are documented here:
https://subliminal.readthedocs.io/en/latest/user/usage.html#cli
are no more available?

I see only these options now:
Options:
  --addic7ed USERNAME PASSWORD    Addic7ed configuration.
  --legendastv USERNAME PASSWORD  LegendasTV configuration.
  --opensubtitles USERNAME PASSWORD
                                  OpenSubtitles configuration.
  --omdb APIKEY                   OMDB API key.
  --cache-dir DIRECTORY           Path to the cache directory.  [default:
                                  /home/dockerprod/.cache/subliminal]

  --debug                         Print useful information for debugging subliminal and for
                                  reporting bugs.

  --version                       Show the version and exit.
  --help                          Show this message and exit.

**Comment by garoto** [comment]
`subliminal download --help`

**Comment by RaXorX** [comment]
I am unsure but did the list_subtitles function ever had a username/password argument option?
When I had posted this https://github.com/Diaoul/subliminal/issues/1008#issuecomment-842973845
message, I had sort of fixed my issue apparently.
`list_subtitles([self.video], {Language(self.language)}, providers=["opensubtitles"], username=["username"], password=["password"])`
I had used the list_subtitles function like that and pretty sure it was getting the subtitles from opensubtitles after that. I don't remember what subliminal/python version I was at earlier, I update my stuff on a regular interval. But after updating python and subliminal both, it doesn't give me the same result. In fact it doesn't even take the username and password arguments at all. 
@garoto @Diaoul
